test/e2e/run_test.go

```diff
@@ -440,28 +440,28 @@ var _ = Describe("Podman run", func() {
 	})
 
 	It("podman run seccomp test", func() {
-		session := podmanTest.Podman([]string{"run", "-it", "--security-opt", strings.Join([]string{"seccomp=", forbidGetCWDSeccompProfile()}, ""), ALPINE, "pwd"})
+		session := podmanTest.Podman([]string{"run", "--security-opt", strings.Join([]string{"seccomp=", forbidGetCWDSeccompProfile()}, ""), ALPINE, "pwd"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).To(ExitWithError())
-		Expect(session.OutputToString()).To(ContainSubstring("Operation not permitted"))
+		Expect(session.ErrorToString()).To(ContainSubstring("Operation not permitted"))
```

Comment:
For anyone understandably not wanting to glaze their eyes looking at all the diffs: this and line 453 are the only two changes that are not `s/"-it", //` (or `-ti`).